src/components/FontAwesomeIcon.js

```diff
@@ -45,23 +45,23 @@ function normalizeIconArgs (icon) {
   }
 }
 
-function FontAwesomeIcon (props) {
+export function FontAwesomeIcon (props) {
```

Comment:
My only complaint with this is that I would consider `src/components/FontAwesomeIcon.js` an internal component. So I would expect there to be a default export in this file. It feels strange to need to do this in the test:

```
import { FontAwesomeIcon } from '../FontAwesomeIcon'
```

Can we switch this to `export default function (props) {`